README.md

```diff
@@ -63,96 +64,103 @@ Please `Star` this repository if you find it valuable and worth maintaining.
 
 ## Quick start
 
-Copy and paste the following code into `build/build.go`:
+Copy and paste the following code into [`build/build.go`](examples/basic/main.go):
 
 ```go
 package main
 
-import "github.com/goyek/goyek"
+import (
+	"fmt"
+	"os"
+
+	"github.com/goyek/goyek"
+)
 
 func main() {
-	flow := &goyek.Taskflow{}
+	if err := os.Chdir(".."); err != nil {
+		fmt.Println(err)
+		os.Exit(goyek.CodeInvalidArgs)
+	}
```

Comment:
We could get rid of this if we make a breaking change by adding this code inside `flow.Main()`.

@dertseha If you agree I will add it here:  https://github.com/goyek/goyek/issues/62

Reply:
I would not put it into main, since I would like to run this also "simply" for a main application - not just libraries, where one might want to keep their go.mod file clean.

This whole cwd/main/script-wrapper thing is for me an indication that something isn't fully fleshed out.
Though, for the current PR, this is OK.

Reply:
Maybe we should add a global parameter to enable changing the working directory? Then the script could  use it and the user could still run it using `go run` from any location

Reply:
I believe this needs a separate discussion, where the requirements and implications are listed (go.mod shall not be polluted -> needs separate mod -> needs to be run in different cwd; multimodule repos; simple out-of-the-box functionality; ...)

Reply:
Tracked under: https://github.com/goyek/goyek/issues/69